sycl/source/CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
   endif()
 
   if (NOT LLVM_ENABLE_ZSTD)
-    target_compile_definitions(${LIB_OBJ_NAME} PRIVATE SYCL_RT_ZSTD_NOT_AVAIABLE)
+    target_compile_definitions(${LIB_OBJ_NAME} PRIVATE SYCL_RT_ZSTD_NOT_AVAILABLE)
```

Comment:
Nit, but still. How do you read code like this?
```
If not ..
 define ZSTD_NOT_AVAILABLE

ifndef ZSTD_NOT_AVAILABLE
```
Not, not, over not, multiplied by not... Applying each `not` slows down my reading abilities by 4x. 

@sergey-semenov, would you mind replace `SYCL_RT_ZSTD_NOT_AVAILABLE` to `SYCL_RT_ZSTD_AVAILABLE` and flip conditions to positive checks, please?

```
if (LLVM_ENABLE_ZSTD)
  define SYCL_RT_ZSTD_AVAILABLE

ifdef SYCL_RT_ZSTD_AVAILABLE
...
```

Reply:
Good point, will do.